doc/releases/changelog-dev.md

```diff
@@ -136,6 +136,25 @@
 
 <h3>Deprecations</h3>
 
+* In-place inversion is now deprecated. This includes `op.inv()` and `op.inverse=value`. Please
+  use `qml.adjoint` instead. Support for these methods will remain till v0.28.
+  [(#2988)](https://github.com/PennyLaneAI/pennylane/pull/2988)
+
+  Don't use:
+
+  >>> v1 = qml.PauliX(0).inv()
+  >>> v2 = qml.PauliX(0)
+  >>> v2.inverse = True
```

Comment:
Some sort of code block is required here (e.g., `pycon` block), rendering if off atm:

![image](https://user-images.githubusercontent.com/24476053/188618085-c5de3d07-29d6-4a43-a121-b8f020ae80dc.png)

Reply:
added.